help/sources/tutorials/ui/create/analytics/rainfocus-experience.md

```diff
@@ -0,0 +1,202 @@
+| exhibitor.externalId      | String   |                1666809514105001lSJN               |           Identifier for exhibitor in client system          |
+| exhibitor.name            | String   |                        IBM                        |                     Name of the exhibitor                    |
+| lead.leadId               | String   |                1666809456617001wyPj               |               RainFocus Identifier of the lead               |
+| lead.note                 | String   |                                                   |                                                              |
+| session.sessionId         | String   |                1666809373585001t4aV               |             RainFocus Identifier for the session             |
+| session.externalId        | String   |                1666809456617001wyPj               |            Identifier for session in client system           |
+| session.code              | String   |                        GS3                        |                        Code of session                       |
+| session.title             | String   |                Inspiration Keynote                |                       Title of session                       |
+| session.length            | Int      |                         90                        |                       Length of session                      |
+| sessiontime.sessiontimeId | String   |                1673033149739001OJLZ               |           RainFocus Identifier for the session time          |
+| sessiontime.startTime     | String   |                2023-03-22 10:00:00                |                      Session start time                      |
+| sessiontime.endTime       | String   |                2023-03-22 10:00:00                |                       Session end time                       |
+| sessiontime.room          | String   |                        B32                        |                       Room for session                       |
+
+To use the sample Schema from RainFocus, you may take one of the the following steps:
+
+* [Create the Schema using the UI](https://experienceleague.adobe.com/docs/experience-platform/xdm/tutorials/create-schema-ui.html?lang=en)
+* [Create the Schema using the API](https://experienceleague.adobe.com/docs/experience-platform/xdm/tutorials/create-schema-api.html?lang=en)
+
+>[!IMPORTANT]
+>
+>* The schema must extend the **XDM ExperienceEvent class.** 
+> * You must ensure that the Schema includes a **primary identity**, and is **enabled for Profile**
+>   * [Define Identity Fields in the UI](https://experienceleague.adobe.com/docs/experience-platform/xdm/ui/fields/identity.html?lang=en)
+>* You may substitute the example identity (Email) for another appropriate identifier such as a sha256 email or ECID.
+
+## Enable the Integration Profile in RainFocus
```

Comment:
for consistent styling, we can remove the numbered steps in this section and just have the steps written out as is. For example for the first one: "Log in to [RainFocus](please add link to RainFocus UI). In the primary navigation, select **Libraries** and **Integration Profiles**."